src/editor/Collaboration.js

```diff
@@ -0,0 +1,56 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var togetherjs = require("togetherjs");
+
+    function Collaboration() {
+        this.togetherjs = togetherjs;
+        var self = this;
+        document.getElementById("collaborate").onclick = function() {
+            self.togetherjs();
+        };
+        this.init();
+    }
+
+    Collaboration.prototype.init = function() {
+        var self = this;
+        self.changing  = false;
+        self.togetherjs.hub.on("data", function(delta) {
+            if(!self.codemirror || self.changing) {
+             return;
+            }
+            self.changing = true;
+            var cm = this.codeMirror;
```

Comment:
Ah finally.
Makes sense why it wasn't working on chrome and was working on Safari.
Thanks for pointing this out.